### DISCUSSION TITLE
RBAC Labels on SSHD nodes

### DISCUSSION THREAD

**Discussion by unreality** [gravitational/teleport#38611] [Q&A] [answered]
Hi, 

Under the [RBAC documentation](https://goteleport.com/docs/server-access/rbac/) it states "Teleport's RBAC system does not extend to OpenSSH servers, as it is not possible to apply labels to servers running sshd. You must enable access to these servers via the Teleport SSH Service instead."

However, via both the CLI and API you **CAN** apply labels to SSHD nodes (the [manual guide](https://goteleport.com/docs/server-access/openssh/openssh-manual-install/#step-15-add-a-node-resource-to-your-teleport-cluster) even does so)

Can these node labels be used for RBAC?

**Comment by webvictim** [comment, marked as answer]
Yes, labels **can** now be used with OpenSSH when registering the nodes with the backend as shown in those docs.

We'll get the out-of-date documentation updated. Thanks for reporting!

**Reply by webvictim** [reply]
Docs were fixed: https://github.com/gravitational/teleport/pull/38618
